jsapp/js/mixins.tsx

```diff
@@ -202,6 +203,7 @@ mixins.dmix = {
   },
 
   _deployAssetFirstTime(asset: AssetResponse) {
+    // TODO: figure out how to switch to toast
```

Comment:
Please create a new GH issue about switching this piece of code to toast. And add a link to it in comment here.

Reply:
Good catch!
It's probably more accurate to say this: 
```suggestion
    // TODO: port this and others from alertify.dialog to new modal system
```

Reply:
Sure :) But regardless, a TODO comment will make us fix it probably never :D Let's make a simple GH issue that says something similar to the comment. The new GH issue can point to https://github.com/kobotoolbox/kpi/issues/3643

Reply:
I noticed this one in particular is a notifier that actually should be ported to toast now. I'll push a commit soon that fixes it, and also updates the other dialog comment to link to the new https://github.com/kobotoolbox/kpi/issues/3977

Reply:
[Did the todo](https://github.com/kobotoolbox/kpi/pull/3972/commits/2f70d76fc8a856536550a615bbc01f508a09d307#diff-a987ee0cbe65021d37b5ba55455a1b551bcfc4a280b5b7cc5e68342545132395L206-R216).
```js
// TODO: figure out how to switch to toast
const deployment_alert = alertify.warning(t('deploying to kobocat...'), 60);
// ...
deployment_alert.dismiss();
```
became
```js
const deployment_toast = notify.warning(t('deploying to kobocat...'), {duration: 60 * 1000});
// ...
toast.dismiss(deployment_toast);
```

[Updated `notify()`](https://github.com/kobotoolbox/kpi/pull/3972/commits/2f70d76fc8a856536550a615bbc01f508a09d307#diff-5530b9ec9efc1ac4fdad31f401e77acfca9c2c57f1123989393e5afb397b150dR28) to accept options (for duration) and return a toastId (for dismissing) to accommodate this use case.